### PR TITLE
Update Wine to v4.17

### PR DESCRIPTION
**Pull request by LeandroStanger**

### CODE REVIEW

File: net.lutris.Lutris.yml [hunk 1]
```diff
@@ -407,8 +407,8 @@ modules:
     sources: &wine_sources
       - type: git
         url: "https://github.com/lutris/wine.git"
-        branch: lutris-4.16
-        commit: d67da44865e4818e6e477a70e7e85dc05186bf96
+        branch: lutris-ge-4.17
```
Comment (gasinvein) [inline net.lutris.Lutris.yml]: Why `-ge-` indeed? Is it the new default for lutris wine?

Comment (LeandroStanger) [reply net.lutris.Lutris.yml]: Not know. @strycore?

**Comment by A6GibKm** [comment]
This is not the master branch and should not be updated. This version is not even suggested as an option in Lutris.

File: net.lutris.Lutris.yml (hunk 1 above)
Comment (tannisroot) [reply net.lutris.Lutris.yml]: No, we don't have a build with such name. Also, wine versions shouldn't be hardcoded anywhere.

Comment (gasinvein) [reply net.lutris.Lutris.yml]: @tannisroot We are bundling Lutris Wine here as a "system" one, just like distributions provide vanilla or staging Wine.

**Pull request closed without merging**